packages/components/src/components/flyout/flyout-styles.ts

```diff
@@ -34,53 +34,73 @@ export const getComponentCss = (
   return getCss({
     '@global': {
       ':host': {
-        display: 'flex',
+        display: 'block',
         ...addImportantToEachRule({
           // needed for correct alignment of the Porsche Grid within the Flyout
           '--pds-internal-grid-outer-column': `calc(${spacingFluidLarge} - ${gridGap})`,
           '--pds-internal-grid-margin': `calc(${spacingFluidLarge} * -1)`,
-          justifyContent: isPositionStart ? 'flex-start' : 'flex-end',
           ...getBackdropJssStyle(isOpen, FLYOUT_Z_INDEX, theme),
           ...colorSchemeStyles,
           ...hostHiddenStyles,
         }),
       },
+      dialog: {
```

Comment:
Aren't this identical to flyout-navigation? If yes, why not extract and share them?

Reply:
Done